sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample5_RecognizePiiEntitiesBatch.cs

```diff
@@ -19,7 +19,9 @@ public void RecognizePiiEntitiesBatch()
             string endpoint = Environment.GetEnvironmentVariable("TEXT_ANALYTICS_ENDPOINT");
             string subscriptionKey = Environment.GetEnvironmentVariable("TEXT_ANALYTICS_SUBSCRIPTION_KEY");
 
-            var client = new TextAnalyticsClient(new Uri(endpoint), subscriptionKey);
+            // Instantiate a client that will be used to call the service.
+            var credentials = new ServiceSubscriptionKey(subscriptionKey);
```

Comment:
Credential should be singular.  You could also new this inside the client constructor to avoid the variable name altogether.